createdatabase.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE community_orgs (
+    id INT PRIMARY KEY,
```

Comment:
I would add `AUTO_INCREMENT` to this as well, it allows the id to be automatically generated to save you some effort later :) https://www.w3schools.com/sql/sql_autoincrement.asp